packages/chart/src/HalfDonut.ts

```diff
@@ -0,0 +1,32 @@
+import { Pie } from "./Pie";
+
+export class HalfDonut extends Pie {
+    constructor() {
+        super();
+    }
+    postUpdate(domNode, element) {
+        super.postUpdate(domNode, element);
+        let x = this._pos.x;
+        let y = this._pos.y;
+        if (this._drawStartPos === "origin" && this._target instanceof SVGElement) {
+            x -= this._size.width / 2;
+            y -= this._size.height / 2;
+        }
+        y += (this._size.height / 4);
+        this._element.attr("transform", "translate(" + x + "," + y + ")scale(" + this._widgetScale + ")");
+    }
+    updateD3Pie() {
+        this.d3Pie
+            .startAngle(-Math.PI / 2)
+            .endAngle(Math.PI / 2)
+            .padAngle(0.0025)
+            .sort(function (b, a) {
+                return a[1] < b[1] ? -1 : a[1] > b[1] ? 1 : 0;
+            })
+            .value(function (d) {
+                return d[1];
+            })
+            ;
```

Comment:
If you called: `super.updateD3Pie()` at the start of this method, you would not need to re-add the `padAngle`, `sort` and `value` calls?